### ISSUE TITLE
SMTP from GoDaddy not working

### ISSUE THREAD

**Issue by darrencoltman** [awesomemotive/WP-Mail-SMTP#96]
## Expected Behavior

Set up to send from GoDaddy smtp to Gmail inbox.

Test email from WP Mail SMTP works fine, straight to inbox, no problem.

Woocommerce sends automatic emails when new order is received, order status changes, order closed, etc.

Email goes first to GoDaddy servers which forwards this email to Gmail. 

DMARC SPF everything set correctly (as shown by successful manual email send from plugin setup screen).

## Current Behavior

However, when sending an automatic email , say, an order receipt, the email goes to Gmail spam folder as before.

As a comparison, woocommerce sends these automatic emails to yahoo mail, etc., works fine, straight to inbox.

**Comment by capuderg** [comment]
Hi @darrencoltman,

is the "Force From Email" checkbox setting enabled in our plugin settings? WooCommerce has a separate "From Email" setting and if you are not forcing the from email address that you are authenticated with on the GoDaddy SMTP server, then the emails could end up in spam.

Just to clarify, you are using the "Other SMTP" mailer in our plugin and are using the GoDaddy SMTP server details, right?

Take care!

**Comment by darrencoltman** [comment]
Hi Gregor,Thanks for replying.
I did try that the first time I installed it (I tried all combinations of all options), but could not get it to work.

I uninstalled it and tried another plugin and did some other things (added spf/dmarc and other stuff) and that plugin eventually worked, although it would not display images.
Anyway, I uninstalled that other plugin and reinstalled your plugin and now it works fine, so whatever I did must have changed something.
Thanks again!Darren.
